### DISCUSSION TITLE
Community

### DISCUSSION THREAD

**Discussion by adepauldami** [serverpod/serverpod#854] [Q&A] [answered]
Am yet to find a community for ServerPod, I intend using ServerPod instead of appwrite if I'm able to find an active community that can help when I need one cause am new to backend technology. 

And currently I'm yet to get a MacBook pro, how do I access the backend UI cause I saw something like it's available only for MacOS for now. I want to be sure of these before switching.

**Comment by vlidholt** [comment, marked as answer]
There is an active discussion board here on Github: https://github.com/serverpod/serverpod/discussions
That being said, we may open a Slack or Discord in the future too.

The Insights app will be available very soon for Windows too! We are currently working on it and it's almost done. :)

**Reply by Katekko** [reply]
Any news about discord server?

**Comment by SandPod** [comment]
For anyone reading this thread, join the [Serverpod Discord Server](https://discord.gg/wJ4pQeHhVc).
